NewForce-at-Mountwest/bangazon-site-the-grave-trolls
# Created Shopping Cart page

# Description

Created a page for the current shopping cart, its contents, and their prices. Adjusted the info displayed on the Products Index and Details views. Removed the "create new" button on the Index page, as this is now accessible via the "Sell Something" link.

## Type of change


# Related Tickets

[Ticket 13.5](https://trello.com/c/5XpTI7VX/22-135-users-should-be-able-to-view-their-current-open-order)

# Testing Instructions

- Clone down these changes on a new branch
- Open the code in Visual Studio
- Adjust the SQLEXPRESS line of AppSettings.json to match the server running on your machine
- Click the IIS Express button (or F5) to run the app
- Log out, if you're logged in
- Ensure the navigation bar is missing most of its options, as added in the last major branch
- Add `/Orders/Details/` to the end of the URL to try to navigate to a cart without being logged in
- This should redirect you to the login page. Log in!
- After logging in, you'll be sent to your shopping cart. If you have no items in your cart, go to the Products page and add a few. Return via the "View Cart" link in the nav bar.
- Ensure the prices appear correctly in your cart, and add up at the bottom.
- NOTE: The Checkout button doesn't work yet! That's the next ticket.
- While looking at your cart, add `/1` to the end of the URL. By doing so, you're trying to view the order of a different user. You should get a 404.

# Checklist:
